package.json

```diff
@@ -73,10 +75,18 @@
     "trezor.js": "6.17.5",
     "uuid": "3.2.1",
     "wallet-address-validator": "0.2.4",
+    "webpack-dev-server": "3.2.1",
     "whatwg-fetch": "2.0.3",
     "zxcvbn": "4.4.2"
```

Comment:
It seems like these new dependencies should only be used in development. If so, they also need to be conditionally disabled in production builds (we should have Webpack set up with `NODE_ENV`).

Reply:
Yes indeed! Correcting right away.